### DISCUSSION TITLE
Куда переехал репозиторий?

### DISCUSSION THREAD

**Discussion by Kirdykun** [Anonym-tsk/nfqws-keenetic#44] [Q&A] [answered]
Пытаюсь обновиться по инструкции, получаю
***
Downloading https://anonym-tsk.github.io/nfqws-keenetic/all/Packages.gz
wget: not an http or ftp url: https://anonym-tsk.github.io/nfqws-keenetic/all/Packages.gz
*** Failed to download the package list from https://anonym-tsk.github.io/nfqws-keenetic/all/Packages.gz
***
Мануал устарел или репозиторий переехал куда?

**Comment by Anonym-tsk** [comment, marked as answer]
Нужно читать мануал с начала, а не с середины
![image](https://github.com/user-attachments/assets/5b7b60ee-173a-4e88-ab1c-a4ba34b806df)

**Reply by Kirdykun** [reply]
Приношу свои извинения, не подумал. Обновилось.
